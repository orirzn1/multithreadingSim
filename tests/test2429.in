L15
S5
O5
N13

T0
I@0x00000000
SUBI $3, $3, 0x3
ADD $0, $0, $4
LOAD $4, $2, 0x0
ADD $2, $0, $1
SUB $6, $7, $1
SUB $6, $0, $2
ADDI $6, $4, 0x2
ADD $3, $7, $0
STORE $6, $5, $2
SUB $3, $4, $6
HALT $0

T1
I@0x00000000
SUB $5, $5, $7
SUBI $0, $4, 0x3
ADDI $4, $5, 0x2
SUB $3, $2, $2
SUBI $7, $1, 0x2
ADDI $4, $4, 0x4
HALT $0

T2
I@0x00000000
ADD $7, $4, $0
SUB $0, $4, $5
ADDI $1, $6, 0x2
LOAD $1, $4, $0
ADDI $1, $6, 0x1
ADD $4, $6, $2
LOAD $3, $6, $2
LOAD $2, $5, $7
STORE $5, $5, 0x0
HALT $0

T3
I@0x00000000
SUBI $1, $3, 0x2
STORE $6, $2, $0
ADD $1, $4, $3
LOAD $7, $4, 0x1
LOAD $2, $2, $0
STORE $0, $1, $7
ADDI $4, $0, 0x4
SUB $1, $0, $0
HALT $0

T4
I@0x00000000
SUB $6, $0, $6
ADDI $4, $7, 0x1
ADDI $2, $7, 0x2
ADDI $5, $4, 0x2
SUB $6, $6, $2
ADDI $1, $5, 0x2
ADD $2, $6, $1
LOAD $7, $6, 0x1
HALT $0

T5
I@0x00000000
SUBI $6, $4, 0x3
ADDI $5, $5, 0x4
STORE $5, $6, 0x3
SUB $7, $3, $0
ADD $1, $3, $2
SUB $2, $6, $2
ADD $7, $0, $6
HALT $0

T6
I@0x00000000
ADD $5, $5, $2
STORE $4, $3, 0x3
SUBI $7, $5, 0x3
SUBI $5, $0, 0x3
ADD $3, $5, $5
ADDI $1, $0, 0x4
LOAD $1, $4, 0x1
ADD $0, $3, $4
ADD $5, $6, $4
ADDI $2, $3, 0x3
HALT $0

T7
I@0x00000000
SUB $0, $5, $1
STORE $3, $3, 0x0
ADD $4, $7, $1
STORE $6, $5, 0x2
SUBI $5, $2, 0x1
HALT $0

T8
I@0x00000000
SUB $3, $4, $2
STORE $5, $7, $3
SUB $5, $6, $2
STORE $0, $1, $2
HALT $0

T9
I@0x00000000
ADD $7, $1, $6
SUBI $2, $4, 0x3
ADD $6, $2, $7
SUB $0, $3, $0
HALT $0

T10
I@0x00000000
SUBI $7, $5, 0x4
SUBI $6, $3, 0x1
STORE $3, $2, $4
ADD $0, $3, $1
SUB $6, $7, $1
HALT $0

T11
I@0x00000000
LOAD $2, $6, $4
SUB $2, $0, $6
LOAD $3, $6, $5
LOAD $1, $4, $3
HALT $0

T12
I@0x00000000
SUBI $6, $3, 0x3
SUBI $1, $6, 0x3
ADDI $7, $1, 0x3
LOAD $6, $3, $7
SUB $0, $3, $0
LOAD $0, $2, $6
SUBI $0, $0, 0x1
SUB $4, $2, $5
STORE $3, $2, 0x0
LOAD $6, $4, 0x1
HALT $0

D@0x0
0x5
0x7
0x7
0x4
0x1
0x8
0x9
0x1
0x9
