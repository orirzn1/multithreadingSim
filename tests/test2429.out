
---- Blocked MT Simulation ----

Register file thread id 0:
	R0 = 0x0	R1 = 0x0	R2 = 0x0	R3 = 0xFFFFFFFE	R4 = 0x5	R5 = 0x0	R6 = 0x7	R7 = 0x0
Register file thread id 1:
	R0 = 0xFFFFFFFD	R1 = 0x0	R2 = 0x0	R3 = 0x0	R4 = 0x6	R5 = 0x0	R6 = 0x0	R7 = 0xFFFFFFFE
Register file thread id 2:
	R0 = 0x0	R1 = 0x1	R2 = 0x0	R3 = 0x0	R4 = 0x0	R5 = 0x0	R6 = 0x0	R7 = 0x0
Register file thread id 3:
	R0 = 0x0	R1 = 0x0	R2 = 0x0	R3 = 0x0	R4 = 0x4	R5 = 0x0	R6 = 0x0	R7 = 0x0
Register file thread id 4:
	R0 = 0x0	R1 = 0x5	R2 = 0x3	R3 = 0x0	R4 = 0x1	R5 = 0x3	R6 = 0xFFFFFFFE	R7 = 0x0
Register file thread id 5:
	R0 = 0x0	R1 = 0x0	R2 = 0xFFFFFFFD	R3 = 0x0	R4 = 0x0	R5 = 0x4	R6 = 0xFFFFFFFD	R7 = 0xFFFFFFFD
Register file thread id 6:
	R0 = 0xFFFFFFFA	R1 = 0x0	R2 = 0xFFFFFFFD	R3 = 0xFFFFFFFA	R4 = 0x0	R5 = 0x0	R6 = 0x0	R7 = 0xFFFFFFFD
Register file thread id 7:
	R0 = 0x0	R1 = 0x0	R2 = 0x0	R3 = 0x0	R4 = 0x0	R5 = 0xFFFFFFFF	R6 = 0x0	R7 = 0x0
Register file thread id 8:
	R0 = 0x0	R1 = 0x0	R2 = 0x0	R3 = 0x0	R4 = 0x0	R5 = 0x0	R6 = 0x0	R7 = 0x0
Register file thread id 9:
	R0 = 0x0	R1 = 0x0	R2 = 0xFFFFFFFD	R3 = 0x0	R4 = 0x0	R5 = 0x0	R6 = 0xFFFFFFFD	R7 = 0x0
Register file thread id 10:
	R0 = 0x0	R1 = 0x0	R2 = 0x0	R3 = 0x0	R4 = 0x0	R5 = 0x0	R6 = 0xFFFFFFFC	R7 = 0xFFFFFFFC
Register file thread id 11:
	R0 = 0x0	R1 = 0x0	R2 = 0x0	R3 = 0x0	R4 = 0x0	R5 = 0x0	R6 = 0x0	R7 = 0x0
Register file thread id 12:
	R0 = 0xFFFFFFFF	R1 = 0xFFFFFFFA	R2 = 0x0	R3 = 0x0	R4 = 0x0	R5 = 0x0	R6 = 0x0	R7 = 0xFFFFFFFD
Blocked MT CPI for this program 2.854369

-----Finegrained MT Simulation -----

Register file thread id 0:
	R0 = 0x0	R1 = 0x0	R2 = 0x0	R3 = 0xFFFFFFFE	R4 = 0x0	R5 = 0x0	R6 = 0x2	R7 = 0x0
Register file thread id 1:
	R0 = 0xFFFFFFFD	R1 = 0x0	R2 = 0x0	R3 = 0x0	R4 = 0x6	R5 = 0x0	R6 = 0x0	R7 = 0xFFFFFFFE
Register file thread id 2:
	R0 = 0x0	R1 = 0x1	R2 = 0x0	R3 = 0x0	R4 = 0x0	R5 = 0x0	R6 = 0x0	R7 = 0x0
Register file thread id 3:
	R0 = 0x0	R1 = 0x0	R2 = 0x0	R3 = 0x0	R4 = 0x4	R5 = 0x0	R6 = 0x0	R7 = 0x0
Register file thread id 4:
	R0 = 0x0	R1 = 0x5	R2 = 0x3	R3 = 0x0	R4 = 0x1	R5 = 0x3	R6 = 0xFFFFFFFE	R7 = 0x0
Register file thread id 5:
	R0 = 0x0	R1 = 0x0	R2 = 0xFFFFFFFD	R3 = 0x0	R4 = 0x0	R5 = 0x4	R6 = 0xFFFFFFFD	R7 = 0xFFFFFFFD
Register file thread id 6:
	R0 = 0xFFFFFFFA	R1 = 0x0	R2 = 0xFFFFFFFD	R3 = 0xFFFFFFFA	R4 = 0x0	R5 = 0x0	R6 = 0x0	R7 = 0xFFFFFFFD
Register file thread id 7:
	R0 = 0x0	R1 = 0x0	R2 = 0x0	R3 = 0x0	R4 = 0x0	R5 = 0xFFFFFFFF	R6 = 0x0	R7 = 0x0
Register file thread id 8:
	R0 = 0x0	R1 = 0x0	R2 = 0x0	R3 = 0x0	R4 = 0x0	R5 = 0x0	R6 = 0x0	R7 = 0x0
Register file thread id 9:
	R0 = 0x0	R1 = 0x0	R2 = 0xFFFFFFFD	R3 = 0x0	R4 = 0x0	R5 = 0x0	R6 = 0xFFFFFFFD	R7 = 0x0
Register file thread id 10:
	R0 = 0x0	R1 = 0x0	R2 = 0x0	R3 = 0x0	R4 = 0x0	R5 = 0x0	R6 = 0xFFFFFFFC	R7 = 0xFFFFFFFC
Register file thread id 11:
	R0 = 0x0	R1 = 0x0	R2 = 0x0	R3 = 0x0	R4 = 0x0	R5 = 0x0	R6 = 0x0	R7 = 0x0
Register file thread id 12:
	R0 = 0xFFFFFFFF	R1 = 0xFFFFFFFA	R2 = 0x0	R3 = 0x0	R4 = 0x0	R5 = 0x0	R6 = 0x0	R7 = 0xFFFFFFFD
Finegrained Multithreading CPI for this program 1.174757

